featuretests/package_test.go

```diff
@@ -51,3 +55,15 @@ func init() {
 func TestPackage(t *stdtesting.T) {
 	coretesting.MgoTestPackage(t)
 }
+
+func runCommand(c *gc.C, args ...string) (*cmd.Context, error) {
+	// Writers need to be reset, because
+	// they are set globally in the juju/cmd package and will
+	// return an error if we attempt to run two commands in the
+	// same test.
+	loggo.ResetWriters()
+	ctx, err := cmd.DefaultContext()
```

Comment:
Yes, the plan is that runJujuCommand will be dropped and this one will be used as a  package-wide standard. I did not want to change all tests to the new version in this PR to minimise the diff. I'll do so in a follow up though.